src/__tests__/recall-by-number.test.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, test } from '@jest/globals'
+import { parsePresetVariableOption } from '../recall-by-number.js'
+import { CompanionOptionValues } from '@companion-module/base'
+import { PresetRecall } from '../camera/commands.js'
+import { prettyBytes } from '../visca/utils.js'
+import { MockContext } from './mock-context.js'
+
+describe('test recall preset values', () => {
+	test('Variable value contant used dec 37', async () => {
```

Comment:
Maybe `'User enters "37" as the preset'`, which I think correlates better with the UI operation being tested.